packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/services/FormVariableViewPreEditingContextProcessor.java

```diff
@@ -46,7 +45,7 @@
 @Service
 public class FormVariableViewPreEditingContextProcessor implements IEditingContextProcessor  {
 
-    public static final String REPRESENTATION_DESCRIPTION_ID = "siriusComponents://representationDescription?kind=formDescription&sourceKind=view&sourceId=c70a4bb6-7310-335d-bda7-a4d1b7c5486f&sourceElementId=e932123d-b916-3537-84d2-86a4f5873d93";
+    public static final String REPRESENTATION_DESCRIPTION_ID = "siriusComponents://representationDescription?kind=formDescription&sourceKind=view&sourceId=c70a4bb6-7310-335d-bda7-a4d1b7c5486f&sourceElementId=f9e6f70f-935f-3b30-9139-640f43de5da7";
```

Comment:
The pattern used in this test will have to be updated to match new tests which do not need to keep such id since it's painful to update when the view model is updated.

Reply:
Ok I'll let you update the test as you want